LayoutTests/fast/dom/no-scroll-when-command-click-fragment-URL.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>Fragment Navigation - ensure opening a fragment URL does not navigate on activating page.</title>
```

Comment:
This title sounds wrong

Reply:
What part is confusing? We need to make sure that when you command-click on a link that it doesn't actually navigate to the fragment on the page that you are currently on.

Reply:
(I explained offline that I was confused by "activating page")